src/lambda/api/PersonaAuthorizer.ts

```diff
@@ -232,6 +232,7 @@ export class PersonaAuthorizer {
         persona_scheme: process.env["PERSONA_SCHEME"],
         persona_port: process.env["PERSONA_PORT"],
         persona_oauth_route: process.env["PERSONA_OAUTH_ROUTE"],
+        cert_background_refresh: false,
```

Comment:
I'd really prefer this be kept separate - as per our guidance on separating functional and refactoring changes. If you have to roll this back, you lose the rest of the PR.

Reply:
Yeah I don't know what I was thinking trying to sneak that in here.